.github/workflows/zip-release.yml

```diff
@@ -0,0 +1,26 @@
+
+name: Publish TriReg2 API as zip file to GitHub Artifacts
+on:
+  release:
+    types: [ created ]
+
+jobs:
+  publish:
+    name: Publish zip to Github Artifacts
+    runs-on: ubuntu-latest
+    steps:      
+      - name: Checkout main repository
+        uses: actions/checkout@v3
+
+      - name: Step 1 - Create a temporary artifact downloads folder
```

Comment:
nitpick: there's no reason to name the steps in order. 

1. Ordering is enforced according to index in the steps list.
2. you forgot the first step : ) 
2. it's prone to manual error
3. if you reorder steps later for whatever reason (or add a step in between) you will have to correct all of them.
4. it adds unnecessary details when inspecting the logs.(because of above)

Reply:
True. 
I have removed the steps in the names since there is only one step now (after having applied the review comments above)